### ISSUE TITLE
font-family-name-quotes error on on sass variables

### ISSUE THREAD

**Issue by TxHawks** [stylelint/stylelint#1239]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug that causes an error when a variable name ends with `font-family`, and its value isn't quoted.

> Which rule, if any, is this issue related to?

`font-family-name-quotes`

> What CSS is needed to reproduce this issue?

```scss
$default-font-family: map-get($font-stacks, default);
```

> What stylelint configuration is needed to reproduce this issue?

Any of the `font-family-name-quotes` rule options

> Which version of stylelint are you using?

`6.3.3`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin via `gulp-postcss`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS variables.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
196:41  ⚠  Expected single quotes around font-family name "map-get($font-stacks, default)" (font-family-name-quotes) [stylelint]
```

**Comment by evilebottnawi** [comment]
@TxHawks show please SCSS code with `font-family`

**Comment by TxHawks** [comment]
@evilebottnawi - 
I did. It's the variable name, which ends with `font-family`, that triggers the error

**Comment by evilebottnawi** [comment]
@TxHawks thanks!

**Comment by evilebottnawi** [comment]
Done in https://github.com/stylelint/stylelint/pull/1242.
